WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsViewController.swift

```diff
@@ -2,6 +2,14 @@ import Foundation
 import UIKit
 import WordPressShared
 
+/// Enumerates the kind of actions available in relation to post subscriptions.
+/// TODO: Add `followConversation` and `unfollowConversation` once the "Follow Conversation" feature flag is removed.
+@objc public enum PostSubscriptionAction: Int {
```

Comment:
Since this is now a standalone enum, I think it's more suitable if the name is updated to `ReaderPostSubscriptionAction`. I'll handle this when I get to tackle this one remaining TODO after your refactor work. 😄

Reply:
And I [moved](https://github.com/wordpress-mobile/WordPress-iOS/pull/17727) it again.  It is now `private` in `ReaderCommentsFollowPresenter`, so you don't need to rename it if you don't want to.